### PR TITLE
Feat/review2

### PR DESCRIPTION
**Pull request by Majorfi**
- Remove all tailwind classnames and default
- Fix some CSS properties
- Fix report 22
- Fix report 9

### CODE REVIEW

**Review by oktapodia — APPROVED** [review]
LGTM

**Review by tcheee — APPROVED** [review]
seems ok to me

File: src/wash/common/Quests.tsx [hunk 1]
```diff
@@ -227,6 +230,28 @@ export function QuestItem(
...
+    justify-content: center;
+    align-items: center;
+    cursor: pointer;
+
+    &:hover {
+      background-color: ${props.questType === 'common'
+        ? colors.pink[700]
+        : colors.cyan[700]};
+    }
+  `;
```
Comment (dennyscode) [inline src/wash/common/Quests.tsx:253]: I would also +1 on the comment above.

File: src/wash/common/icons/IconArrowLeft.tsx [hunk 2]
```diff
@@ -8,25 +8,25 @@ export function IconArrowLeft(
   return (
     <svg
       {...props}
-      width="12"
-      height="12"
-      viewBox="0 0 12 12"
-      fill="none"
-      xmlns="http://www.w3.org/2000/svg"
+      width={'12'}
+      height={'}12'}
```
Comment (dennyscode) [inline src/wash/common/icons/IconArrowLeft.tsx:12]: I guess it´s a typo here.

**Review by dennyscode — CHANGES_REQUESTED** [review]
I tried to build locally and it failed. Please take another look at that issue and make sure the build runs smoothely. Maybe, also remove those "unused variables" introduced in wash-components. Although, we have a bunch of that to clean up already, let´s try to satisfy our linter as much as we can 😄
image

Thanks for the hard work!

**Pull request merged**